.chloggen/promote-prwreceiver-alpha.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: prometheusremotewritereceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Promote prometheusremotewritereceiver to alpha
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [41034]
```

Comment:
It fixes this one, right? https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/37277

Reply:
Right. I did so many pr's that I got lost in what initial problem we were solving kkk